web-components/src/components/cards/card.stories.tsx

```diff
@@ -101,7 +101,7 @@ export const projectCard = (): JSX.Element => (
         sellInfo: {
           creditsAvailable: 0,
           creditsAvailableForUser: 1190,
-          pricePerTon: '17.20-24.20',
+          avgPricePerTonLabel: '17.20-24.20',
```

Comment:
probably we can change the values here as well so it's more realistic now that is not a range anymore